package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "lecture-08-node-api",
+  "version": "1.0.0",
+  "description": "401 JS --  Lab 08 rest api using http module ===",
+  "main": "index.js",
+  "scripts": {
+    "test": "./node_modules/mocha/bin/mocha"
```

Comment:
Good job including a test script. Now try to get in the habit of including your ``watch`` script to so that anyone can just run Nodemon.